src/main/java/org/alliancegenome/curation_api/model/entities/Reference.java

```diff
@@ -49,7 +51,8 @@ public class Reference extends InformationContentEntity {
 
 	@IndexedEmbedded(includeDepth = 1)
 	@IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
-	@ManyToMany
+	@OneToMany
```

Comment:
The reason for this change as effectively onetomany and manytomany here are going to be the same it should stay as a manytomany as that describes the relationship between the two tables.

Reply:
If you are trying to make sure that the cross refs are always loaded then put a fetch = EAGER on the @ManyToMany

Reply:
It's not many-to-many though - we don't reuse/share xrefs.

Reply:
yes that may be the case, but the relationship doesn't change, its more of a policy in the code. if it was truly a oneto many the a referenceId would need to be on the cross reference and its not so from a object to object point of view a cross reference can be on many data types and a data type can have many cross references just just happen to enforce a single entry in the list

Reply:
Ok, makes sense - will revert to @ManyToMany